packages/js-client/src/internal/modules/decoding.ts

```diff
@@ -0,0 +1,12 @@
+import { ISimpleStorageClientDecoding } from '../interfaces';
+import { ClientCore } from '@aragon/sdk-client-common';
+
+export class SimpleStorageClientDecoding
+  extends ClientCore
+  implements ISimpleStorageClientDecoding
+{
+  // implementation of the methods in the interface
+  public myAction() {
+    console.warn('Method not implemented.');
+  }
```

Comment:
As said before, I would provide functional code doing real things. With a big comment telling people where to implement their own logic

Reply:
Mmm okay, I caan implement the functions for the Simple storage plugin